### PR TITLE
Remove doc about the removed command shouldafound

### PR DESCRIPTION
**Pull request by aryx**
This goes with https://github.com/returntocorp/semgrep/pull/8493

test plan:
nope

# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/reporting-false-negatives.md [hunk 1]
```diff
@@ -7,13 +7,10 @@ hide_title: true
...
-import IntroShouldafound from "/src/components/concept/_shouldafound.mdx"
-import ProcedureShouldafound from "/src/components/procedure/_shouldafound.mdx"
 
-# Reporting false negatives with `shouldafound`
+# Reporting false negatives
 
-<IntroShouldafound />
-
-<ProcedureShouldafound />
+Just submit an issue on the semgrep issue tracker here: https://github.com/returntocorp/semgrep/issues
```
Comment (s-santillan) [inline docs/reporting-false-negatives.md:13]:
```suggestion
Submit an issue on the [Semgrep issue tracker](https://github.com/returntocorp/semgrep/issues).
```

File: docs/reporting-false-negatives.md [hunk 2]
```diff
@@ -7,13 +7,10 @@ hide_title: true
...
+It's usually better to include a playground link with a rule and some test code illustrating the false negative.
```
Comment (s-santillan) [inline docs/reporting-false-negatives.md:14]:
```suggestion
If applicable, include a [Semgrep Playground](https://semgrep.dev/playground) link with a **rule** and some **test code** illustrating the false negative.
```

**Review by s-santillan — APPROVED** [review]
Thank you for updating the docs, @aryx. Approved with minor suggestions that you can commit with a single click if they work for you.
